unit_test.sh

```diff
@@ -16,6 +16,11 @@ DATABASE_USER="compliance"
 DATABASE_PASSWORD="changeme"
 DATABASE_NAME="compliance-test"
 
+# if this is a PR, use a different tag, since PR tags expire
+if [ -n "$ghprbPullId" ]; then
```

Comment:
Can this be moved closer where the `IMAGE_TAG` is being set? There are other variables that depend on this value.

Nit: use bash test `[[ ]]`.

Reply:
There are already two places where a `IMAGE_TAG` is defined, line 5 and 14. In between other vars use `IMAGE_TAG`. To which one of these should it be closer to?

Reply:
I'd move it right under line 6, after the initial `IMAGE_TAG` definition.

Reply:
@vkrizan But then it would be defined again on line 14 and probably cause issues again.